src/main/java/de/tum/in/www1/artemis/web/rest/dto/FileUploadAssessmentDTO.java

```diff
@@ -2,7 +2,10 @@
 
 import java.util.List;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+
 import de.tum.in.www1.artemis.domain.Feedback;
 
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
 public record FileUploadAssessmentDTO(List<Feedback> feedbacks, String assessmentNote) {
 }
```

Comment:
Failing test: de.tum.in.www1.artemis.exercise.fileupload.FileUploadAssessmentIntegrationTest#testSaveFileUploadAssessment_studentHidden

I think the corresponding Resource method does not expect `feedbacks` to be `null`, but gets exactly that because `feedbacks` is sent as an empty list. Can you have a look?